src/window.rs

```diff
@@ -700,6 +719,12 @@ pub trait WinHandler {
     #[allow(unused_variables)]
     fn idle(&mut self, token: IdleToken) {}
 
+    /// Called when the AccessKit adapter receives an action request from a client
+    /// (e.g. screen reader).
+    #[allow(unused_variables)]
+    #[cfg(feature = "accesskit")]
+    fn accesskit_action(&mut self, request: accesskit::ActionRequest) {}
```

Comment:
Yes; I figured since the `accesskit` feature is optional and not enabled by default in glazier, it would be better to not have multiple levels of optionality.